requirements.txt

```diff
@@ -33,7 +33,8 @@ python-novaclient
 python-octaviaclient
 python-swiftclient
 tenacity
-distro-info
+#see https://github.com/openstack-charmers/zaza/issues/456
+#distro-info
```

Comment:
I think we can just remove these two lines.  It is thought that distro-info isn't actually used anywhere, and if a particular charm needs it, it can add an override.

Reply:
Addressed. Resolving conversation.